app/controllers/profiles_controller.rb

```diff
@@ -223,7 +223,7 @@ def profiles_for_index
         .by_region(current_region)
         .includes(:translations)
         .main_topic_translated_in(I18n.locale)
-        .random
```

Comment:
Are the profiles then sorted by id?  If so, then perhaps we could display the highest (i.e. newest) first?

Reply:
sorting is a bit random now, since we do not tell them how to sort. If that is ok we can sort by newest created.